india_compliance/gst_india/overrides/test_transaction_data.py

```diff
@@ -0,0 +1,197 @@
+import re
+
+import frappe
+from frappe.tests.utils import FrappeTestCase
+from frappe.utils import add_to_date, getdate
+from frappe.utils.data import format_date
+
+from india_compliance.gst_india.utils.tests import create_sales_invoice
+from india_compliance.gst_india.utils.transaction_data import (
+    GSTTransactionData,
+    validate_non_gst_items,
+)
+
+
+class TestTransactionData(FrappeTestCase):
+    @classmethod
+    def setUpClass(cls):
+        frappe.db.savepoint("before_test_transaction_data")
+
+    @classmethod
+    def tearDownClass(cls):
+        frappe.db.rollback(save_point="before_test_transaction_data")
+
+    @classmethod
+    def setUp(cls):
+        pass
```

Comment:
Not needed.

Also add item_tax_details

Reply:
item_tax_details is already called by function `get_item_details`. Added case for `get_address_details`